src/lib/utils/meta_tables/meta_segments_table.hpp

```diff
@@ -5,9 +5,10 @@
 namespace hyrise {
 
 /**
- * This is a class for showing information of all stored segments via a meta table.
- * Here, we do not provide the distinct value count per segment.
- * For slower results with that information use MetaSegmentsAccurateTable.
+ * This is a class for showing information of all stored segments via a meta table. Here, we only provide
+ * - the distinct value count per segment if it is cached by a statistics object and
+ * - an estimated memory usage derived by sampling.
+ * For slower results where this information is more accurate use MetaSegmentsAccurateTable.
```

Comment:
Reword.

Reply:
`MetaSegmentsAccurateTable provides similar information, but computes uncached distinct value counts and calculates the memory usage from all values.`?

Reply:
`MetaSegmentsAccurateTable provides more accurate information, but can be slower to access. It computes uncached distinct value counts and calculates the memory usage from all values.`
?